pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -644,15 +628,13 @@ cdef class {{name}}HashTable(HashTable):
             int ret = 0
             {{c_type}} val, na_value2
             khiter_t k
-            {{name}}VectorData *ud
```

Comment:
Something very strange was going on here, where changing this to a non-pointer and assigning `ud = uniques.data` then trying to pass that via reference to `needs_resize` was failing. I think has something to do with Cython pointer/reference semantics, but rather than dive too deep figured it was trivial enough to just remove